packages/key-management/src/cip8/cip30signData.ts

```diff
@@ -50,6 +50,11 @@ const getAddressBytes = (signWith: Cardano.PaymentAddress | Cardano.RewardAccoun
 };
 
 const getDerivationPath = async (signWith: Cardano.PaymentAddress | Cardano.RewardAccount, keyAgent: AsyncKeyAgent) => {
```

Comment:
I think it would be better to add DrepID in the function signature:
```
- const getDerivationPath = async (signWith: Cardano.PaymentAddress | Cardano.RewardAccount, keyAgent: AsyncKeyAgent) => {
+ const getDerivationPath = async (
+  signWith: Cardano.PaymentAddress | Cardano.RewardAccount | Cardano.DRepID,
+  keyAgent: AsyncKeyAgent
+ ) => {